libcob/common.h

```diff
@@ -1645,6 +1645,7 @@ COB_EXPIMP int		cob_last_exception_is	(const int);
 
 COB_EXPIMP int		cob_last_exit_code	(void);
 COB_EXPIMP const char*	cob_last_runtime_error	(void);
+COB_EXPIMP char*        cob_get_strerror (void);
```

Comment:
The (purpose of the) exteralization is missing. Note that this only can be reasonably called if the errors came from libcob, if not then we need to mirror this for example in cobc.

Reply:
This change is needed to get the correct error message, for example in the profiling PR when `fopen` fails.